### ISSUE TITLE
Document Configuring JMX Adapter

### ISSUE THREAD

**Issue by dboreham** [jolokia/jolokia#471]
Apologies if this is document somewhere already. I'm interested in using the code in [this PR](https://github.com/rhuss/jolokia/pull/431) to connect jconsole to my jolokia-enabled production JMVs. 
I'm lacking in deep JMX knowledge, but I get the impression that the idea here is to load a transport provider plugin class, or whatever is the correct terminology here, into a JMX client tool such as jconsole, such that it speaks to its target JVM using the jolokia protocol rather than stock JMX protocol (RMI over TCP?). This sounds great, but I have no idea how to a) get that code in place so the JMX tool can load it nor b) how to configure said tool to use the new transport. The YouTube videos hint that there is a URI scheme to target a specific transport (jolokia), then subsequent URI path is the jolokia URL. I'd be happy to write this doc if someone can provide me with some hints as to how to do the config. Thanks.

**Comment by rhuss** [comment]
good point, and a comprehensive documentation is definitely on the task list. In the meantime, could you give some hints about the usage of the connectors, @skarsaune ? We could use that then directly as input for the proper documentation.

**Comment by skarsaune** [comment]
Absolutely. Sorry, have been quite busy. It is in my list. Will reply with example soon

**Comment by skarsaune** [comment]
So for instance JConsole as the simplest tool, you could simply add jolokia to the classpath as shown in this script (modivy version 1.7.0 and location where you have downloaded the jar)
https://github.com/rhuss/jolokia/blob/master/client/jmx-adapter/src/test/scripts/runJConsole.sh

**Comment by skarsaune** [comment]
@dboreham : Feel free to try out the plugins for Java Mission Control. Let me know how it works for you. 
Installation instructions in the Readme : https://github.com/skarsaune/jmc-cloud-extensions
